appstream.patch

```diff
@@ -0,0 +1,55 @@
+From 45a7eb2788bef6de0ffe0f4fd49f29226c961c1f Mon Sep 17 00:00:00 2001
+Message-Id: <45a7eb2788bef6de0ffe0f4fd49f29226c961c1f.1691951179.git.matthias.mailaender@lablicate.com>
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?=
+ <matthias.mailaender@lablicate.com>
+Date: Sun, 13 Aug 2023 18:31:43 +0200
+Subject: [PATCH] Update AppStream metadata.
+
+---
+ asc.appdata.xml.in | 19 ++++++++++++++++---
+ 1 file changed, 16 insertions(+), 3 deletions(-)
+
+diff --git a/asc.appdata.xml.in b/asc.appdata.xml.in
+index b00f2e2eb..a60e6fe3f 100644
+--- a/asc.appdata.xml.in
++++ b/asc.appdata.xml.in
+@@ -1,7 +1,10 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+-<application>
++<component type="desktop">
+     <id type="desktop">asc.desktop</id>
```

Comment:
the ID is never changed (sadly) by flatpak-builder. So this will fail. See my other comment about properly submitting the changes with the app-id uptream.